extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -25,7 +23,6 @@
     --option-2-percentage: 0%;
     --option-3-percentage: 0%;
     --option-4-percentage: 0%;
-    font-family: 'Poppins-bold', sans-serif !important;
```

Comment:
Poppins-bold doesn't exist, so Arial was being used instead

Reply:
Can we change update the font import so we also get `bold`? Then we can set the `font-weight` to `700` (bold). (`400` is the default)

https://fonts.google.com/specimen/Poppins

```
https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap
```